### ISSUE TITLE
Incompatible with Regions Unexplored in 1.20.1

### ISSUE THREAD

**Issue by bv7dev** [ValkyrienSkies/Valkyrien-Skies-2#616]
### Mod Name

Regions Unexplored Fabric + TerraBlender (dependency)

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Fabric

### Issue description

Crashes on launch with error message:

The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'regions_unexplored'!

### Issue reproduction

![image](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/64686444/f9bf97c6-f2ea-4265-985f-48f12a1c6d19)

### Logs

[latest.log](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/files/13229541/latest.log)

**Comment by FirstMegaGame4** [comment]
I found a temporary solution to it, you should open ValkyrienSkies jar and search for an `org/apache/common/lang3` directory and delete it. It's an included package that should not be here replacing the `ArrayUtils` class used by the `CompleteConfig` dependency of `RegionsUnexplored`.

**Comment by tomcrusade** [comment]
Just fo anyone that encounters different, but similar bug to this.

I also found an error when running "extended block shapes - blockus" mod with any valkyrien-skies-2 version of 1.20.1 with message:

```

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'extshape'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:458)
	at net.minecraft.client.main.Main.main(Main.java:211)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NoSuchMethodError: 'java.lang.Object org.apache.commons.lang3.ObjectUtils.getIfNull(java.lang.Object, java.util.function.Supplier)'
	at pers.solid.extshape.block.ExtShapeFenceGateBlock.<init>(ExtShapeFenceGateBlock.java:31)
	at pers.solid.extshape.builder.FenceGateBuilder.lambda$new$0(FenceGateBuilder.java:15)
	at pers.solid.extshape.builder.AbstractBlockBuilder.createInstance(AbstractBlockBuilder.java:298)
	at pers.solid.extshape.builder.AbstractBlockBuilder.build(AbstractBlockBuilder.java:314)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at pers.solid.extshape.builder.BlocksBuilder.build(BlocksBuilder.java:408)
	at pers.solid.extshape.block.ExtShapeBlocks.<clinit>(ExtShapeBlocks.java:107)
	at pers.solid.extshape.ExtShape.onInitialize(ExtShape.java:50)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
	... 6 more
```

After removing `org/apache/common/lang3` these mods work properly

**Comment by KirboSoftware** [comment]
Hey, the fabric maintainer of regions unexplored here. I'm working on an alternative config library from scratch to resolve this issue. Sorry for bothering you guys with this, sadly. I have most of the content of lib almost done and then will the implantation process begin. So the next fabric version should have this fix 👍 

Once again, sorry for bothering you guys with this.

**Comment by Poopooracoocoo** [comment]
To expand on what @FirstMegaGame4 you can open jar files in 7-zip.

**Comment by thetheaplant** [comment]
Fixed in #798.
